Makefile

```diff
@@ -58,12 +58,29 @@ test-e2e-polybft:
 
 .PHONY: run-local
 run-local:
-	$(MAKE) compile-core-contracts
 	docker-compose -f ./docker/local/docker-compose.yml up -d --build
 
 .PHONY: stop-local
 stop-local:
-	docker-compose -f ./docker/local/docker-compose.yml stop
+	EDGE_CONSENSUS=ibft docker-compose -f ./docker/local/docker-compose.yml stop
+
+.PHONY: destroy-local
+destroy-local:
+	EDGE_CONSENSUS=ibft docker-compose -f ./docker/local/docker-compose.yml down -v
+
+.PHONY: run-local-polybft
+run-local-polybft:
+	$(MAKE) compile-core-contracts
+	EDGE_CONSENSUS=polybft docker-compose -f ./docker/local/docker-compose.yml up -d --build
+
+.PHONY: stop-local-polybft
+stop-local-polybft:
+	EDGE_CONSENSUS=polybft docker-compose -f ./docker/local/docker-compose.yml stop
+
+.PHONY: destroy-local-polybft
+destroy-local-polybft:
+	EDGE_CONSENSUS=polybft docker-compose -f ./docker/local/docker-compose.yml down -v
+
```

Comment:
IMO Makefile is not the place to manage the lifecycle of local env, it's not building any aspect of the binary, this belongs to a `docker` script in `scripts` folder

Reply:
This local env var determines which consensus mechanism is to be used. It acts as a kind of a switch statement.   
The `docker-compose` file passes it to the `Dockerfile` as a second argument, which is used by the `entrypoint` script to create genesis for a specific consensus.
It must be set before the `init` container spins up, as it needs to know which consensus is being used.

The alternative to this approach is to have separate files for each consensus.    
We would than need to have dedicated `docker-compose`, `Dockerfile` and `entrypoint` script for each consensus.

IMHO, this is much simpler approach, as we're using the same files and the users can easily make consensus selection just by setting `EDGE_CONSENSUS` env var.

But if you disagree with this, we can easily split this logic.

Reply:
@vcastellm updated readme file with the instruction on how to run with `make` or directly with `docker-compose`.
Kept the same logic, as we're using existing docker files, but just switching the consensus with `EDGE_CONSENSUS` env var. This keeps the scripts simple and lean.     
Also added in some customization options for the chain.

Reply:
My proposal is to remove all the `run-local` stuff out from Makefile to a `docker` script in `scripts`, the fact that it's driven by env variables it's not related to using Makefile, it can be also improved by adding an option to the `cluster` script for running under docker, `scripts/cluster polybft --docker` should be the UI

Reply:
@vcastellm I see now that we already have dedicated `scripts` folder. Edited the `cluster` script to accommodate `ibft` and `polybft` docker environments.     
Let me know if this works for you and the rest of the team.

Will change the PR description, but there are `README.md` files in both `docker` and `scripts` folder with some explanation how to use the script.